docker/Dockerfile

```diff
@@ -14,12 +14,17 @@ RUN go get -v github.com/readium/readium-lcp-server/lcpencrypt
 
 FROM phusion/baseimage:focal-1.1.0 As base
 
+COPY ./docker/simplified_app.sh /ls_build/
```

Comment:
The files in the docker folder don't change very often. You should be able to still do 

```dockerfile
COPY ./docker /ls_build
RUN /bin/bash -c "/ls_build/simplified_app.sh \
      && /ls_build/logrotate.sh"
``` 

You can remove the parts of `simplified_app.sh` that depend on `poetry.lock` & `pyproject.toml` and put them into a different script.